ML-for-S2S/gan.yml

```diff
@@ -0,0 +1,65 @@
+seed: 1000
+save_loc: "/glade/work/schreck/repos/ML-for-S2S/ML-for-S2S"
+
```

Comment:
Some comments about what some of these variables mean would be really helpful :)